Mathlib/Data/Complex/Module.lean

```diff
@@ -472,3 +472,24 @@ theorem imaginaryPart_smul (z : ℂ) (a : A) : ℑ (z • a) = z.re • ℑ a +
 #align imaginary_part_smul imaginaryPart_smul
 
 end RealImaginaryPart
+
+section Rational
+
+open Cardinal Module
+
+@[simp]
+lemma Real.rank_rat_real : Module.rank ℚ ℝ = continuum := by
+  refine (Free.rank_eq_mk_of_infinite_lt ℚ ℝ ?_).trans mk_real
+  simpa [mk_real] using aleph0_lt_continuum
+
+@[simp]
+lemma Complex.rank_rat_complex : Module.rank ℚ ℂ = continuum := by
```

Comment:
If already in the `Real` (ditto Complex) namespace, do we need `_real` at the end?

Reply:
Yeah, `Complex.rank_rat` or `Module.rank_rat_complex` both seem like better names. What's the lemma that `Module.rank ℚ ℝ = 2` called?

Reply:
yes, you found why I named it the way I did, but I see your points.